tests/test_work_unit.py

```diff
@@ -93,6 +121,9 @@ def test_save_and_load_fits(self):
                     for x in range(p1.get_dim()):
                         self.assertAlmostEqual(p1.get_value(y, x), p2.get_value(y, x))
 
+                # No per-image WCS
```

Comment:
Can we test generating a per-image WCS as well?

Reply:
Good call. I added a mix of WCS and non-WCS to test the parsing functions.